.github/workflows/tax-education-load-tests.yml

```
@@ -0,0 +1,228 @@
+      - name: Set up Python
+        uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
+        with:
+          python-version: "3.11"
+          cache: pip
+
+      - name: Install test dependencies
+        run: pip install httpx
+
+      - name: Determine phase inputs
+        id: params
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:10:34: Double quote to prevent globbing and word splitting

(shellcheck)